esp/esp/program/models/__init__.py

```diff
@@ -1067,18 +1067,28 @@ def getShirtInfo(self):
         if 'class_approved' in teacher_dict:
             query = teacher_dict['class_approved']
             query = query.filter(registrationprofile__most_recent_profile=True)
-            query = query.values_list('registrationprofile__teacher_info__shirt_type',
-                                      'registrationprofile__teacher_info__shirt_size')
-            query = query.annotate(people=Count('id', distinct=True))
+            if Tag.getBooleanTag('teacherinfo_shirt_type_selection') == False:
```

Comment:
Since this is a boolean now, you can just do `if not Tag.getBooleanTag(...):`, no need to compare to `False`.